tools/misc/rdiff-backup-wrap

```diff
@@ -35,14 +35,20 @@ case "$SOURCE" in
 esac
 
 # Pick the right version of rdiff-backup
-if [ $REMOTE_SOURCE -eq 0 ]; then
-    CMD="rdiff-backup2"
-else
+CMD=rdiff-backup	# default if nothing else is found
+
+for cmd in rdiff-backup-1 rdiff-backup2
+do
+	if command -v $cmd 2>&1 > /dev/null
+	then
+		CMD=$cmd
+	fi
+done
+
+if [ $REMOTE_SOURCE -eq 1 ]; then
     if rdiff-backup --test-server "$SOURCE" 2>&1 > /dev/null; then
         CMD="rdiff-backup"
-    else
-        CMD="rdiff-backup2"
     fi
 fi
 
-"$CMD" $@
\ No newline at end of file
+"$CMD" $@
```

Comment:
```suggestion
"$CMD" "$@"
```
See man-page of bash and search for `$@`...

Reply:
https://man7.org/linux/man-pages/man1/bash.1.html#PARAMETERS -> Special parameters

Reply:
Tossing up should I also "exec" this, rather than just running it as a separate process?  What do you think?

Reply:
Indeed I personally prefer to use `exec` for wrapper scripts, but it's more a personal preference (the overhead of bash is limited compared to the footprint of rdiff-backup).